### PR TITLE
8.0 FIX Wrong taxes and account when partner has a FP, but FP=false on SO

### PR DESCRIPTION
**Pull request by alexis-via**
The related PR on Odoo with a detailed explaination is available here:

https://github.com/odoo/odoo/pull/11409

One of my French users found this bug with the following scenario: one of his regular customers based in Algeria took the opportunity of a trip in France to come to the factory in France and order some good on site ; as it is a "local" order, we have to invoice with VAT.

This is a important bug because it has an impact on accounting (wrong account selected).

### CODE REVIEW

**Comment by alexis-via** [comment]
I don't understand why this PR took a commit form simongoffin... and not only my 2 commits.

**Comment by pedrobaeza** [comment]
This PR includes undesired commits I think

**Comment by alexis-via** [comment]
@pedrobaeza yes, and I don't understand why, as I said in my previous comment.

**Comment by alexis-via** [comment]
Mmm, I think I understand: this commit of simongoffin is present in odoo but has not been replayed yet in OCB...

**Comment by pedrobaeza** [comment]
If you use the same branch then that's the problem. Tonight everything will be solved.

**Comment by alexis-via** [comment]
Is it a problem to use the same branch ? Should I use 2 different branches ?
Using the same branch is a big time saver when contributing on OCB...

**Comment by pedrobaeza** [comment]
No, it can be, but until this night it will be this "de-synchronization"

File: addons/account_asset/account_asset_view.xml [hunk 1]
```diff
@@ -161,7 +161,7 @@
                                     </group>
                                 </form>
                             </field>
-                            <button type="object" name="compute_depreciation_board" string="Compute" icon="terp-stock_format-scientific" colspan="2" attrs="{'invisible':[('state','=','close')]}"/>
+                            <button type="object" name="compute_depreciation_board" string="Compute" icon="terp-stock_format-scientific" colspan="2" invisible="1"/>
```
Comment (StefanRijnhart) [inline addons/account_asset/account_asset_view.xml:164]: ~~Change is unrelated, I believe.~~ old news.

**Comment by StefanRijnhart** [comment]
Already commented on the code, but the unrelated change is still here. I'll check the synchronization.

**Comment by StefanRijnhart** [comment]
Well, the code is in OCB (https://github.com/OCA/OCB/blob/8.0/addons/account_asset/account_asset_view.xml#L164) and so is the commit. This must be a git glitch. @alexis-via could you rebase on any earlier Odoo commits?

**Comment by alexis-via** [comment]
It seems to me that all these problems are related to the fact that I used the same branch to do my PR on odoo and my PR on OCB. So i prepared a new branch based on OCB with a new PR and I cancel this PR.

**Comment by alexis-via** [comment]
The new OCB PR is here: https://github.com/OCA/OCB/pull/472
